server/schedule/filters.go

```diff
@@ -349,3 +349,71 @@ func (f rejectLeaderFilter) FilterSource(opt Options, store *core.StoreInfo) boo
 func (f rejectLeaderFilter) FilterTarget(opt Options, store *core.StoreInfo) bool {
 	return opt.CheckLabelProperty(RejectLeader, store.Labels)
 }
+
+// StoreStateFilter is used to determine whether a store can be selected as the
+// source or target of the schedule based on the store's state.
+type StoreStateFilter struct {
```

Comment:
Is that better to keep consistent with the other filters?

Reply:
em, how should I change it to 'keep consistent'?

Reply:
Maybe you can provide something like `NewStoreStateFilter`?